librz/analysis/var_global.c

```diff
@@ -64,7 +63,9 @@ RZ_API RZ_OWN bool rz_analysis_var_global_add(RzAnalysis *analysis, RZ_NONNULL R
 	}
 
 	global_var->analysis = analysis;
-	global_var->flag_item = rz_flag_set(global_var->analysis->flb.f, global_var->name, global_var->addr, rz_type_db_get_bitsize(global_var->analysis->typedb, global_var->type) / 8);
+	rz_flag_space_push(global_var->analysis->flb.f, "globals");
+	rz_flag_set(global_var->analysis->flb.f, global_var->name, global_var->addr, rz_type_db_get_bitsize(global_var->analysis->typedb, global_var->type) / 8);
```

Comment:
shall we check this as well and fail to create the global var if a flag cannot be created?

Reply:
same as above